### DISCUSSION TITLE
add more compile doc on windows

### DISCUSSION THREAD

**Discussion by RyanCode** [protobuf-c/protobuf-c#565] [Q&A]
It's too hard for me who have not used Windows for development to compile protobuf-c on Windows

**Comment by danyeaw** [comment]
Hi @RyanCode, while it doesn't totally replace the need for better docs, you can build Protobuf-c with [gvsbuild](https://github.com/wingtk/gvsbuild). Getting a development environment setup in Windows and all the dependencies compiled isn't trivial, and gvsbuild helps with this. Once you have followed the setup instructions, you can build it with `gvsbuild build protobuf-c`.
